src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4722,7 +4722,11 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
             // Set maximum supply air temperature for supplemental heating coil
             state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxSATFromSuppHeatCoil = rNumericArgs(11);
             // set maximum outdoor dry-bulb temperature for supplemental heating coil operation
-            state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxOATSuppHeatingCoil = rNumericArgs(12);
+            if (NumNums < 12) {
+                state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxOATSuppHeatingCoil = 21.0;            
+            } else {
+                state.dataHVACVarRefFlow->VRFTU(VRFTUNum).MaxOATSuppHeatingCoil = rNumericArgs(12);                        
+            }
```

Comment:
sets the default value since the `min-Field` does not include this input field.

Reply:
Without this change the when the `NumNums < 12` is true `0.0` will be default value, which is the initialized value for `rNumericArgs` variable.